src/shared/defaultAwsContextCommands.ts

```diff
@@ -150,7 +145,20 @@ export class DefaultAWSContextCommands {
                     secretKey: state.secretKey,
                 })
 
-                return state.profileName
+                const sharedProviderId: CredentialsId = {
+                    credentialSource: SharedCredentialsProvider.getProviderType(),
+                    credentialTypeId: state.profileName,
+                }
+
+                window.showInformationMessage(
+                    localize(
+                        'AWS.message.prompt.credentials.definition.done',
```

Comment:
Doesn't look like this is in package.nls.json

Reply:
Does not need to be in `package.nls.json`
The real localization will extract it out.

Reply:
The pattern I have been following since an earlier PR in which I received feedback is to only add explicit NLS file entries when adding references in `package.json`.